schemas/validation_tests/sample-valid-1.json

```diff
@@ -0,0 +1,84 @@
+{
+    "object_id": {
+        "alias" : "My_10samples_X_from_NM305004", 
+        "center_name" : "EBI-TEST",
+        "ega_accession": "EGAN00000000001"
+    },
+    "object_title": "Buccal swab from COVID+ patient NM305004",
+    "object_description": "Buccal swab from COVID positive patient (NM305004) was taken on a sunny morning, had a lower volume than expected, then was sent to our labs in California",
+    "organism_descriptor": {
+        "taxon_id": "9606",
+        "scientific_name": "homo sapiens",
+        "common_name": "human"
+    },
```

Comment:
Since the ``individual`` *must* be a human (or so I imagined), I did not include the organism descriptor node within its schema. Apart from the most common case, where a tissue/swab sample is taken from the individual, I thought that having the organism description within the sample would be beneficial for cases where the sample derives from the individual but its content is not part of it. For instance: viral samples, microbiota, xenografts...

Reply:
In the context of EGA, yes the individual will (almost!) always be human. There is actually some non-human data in EGA (I think from a very long time ago). My proposal is to have this block in both the individual and sample schema. For the reason you mention (a sample from a human might be non-human) and for the reason I mention (someone using/accessing EGA sample/individual metadata _not_ in the context of EGA, and thus they won't necessarily know that the individual is likely human).

Reply:
As far as I remember from talking with Aina, non-human related samples from a very long time ago are the only ones we hold, and we explicitly state now to only accept human-related samples. Now, I agree that having the organism node may be useful to know what the individual is, but just to have the least amount of redundant information: 

- Shall I add the whole organism descriptor node to the individual or just some parts (e.g. the ``taxon_id`` alone)? 
- Would there be an scenario in which we will receive data unrelated to humans as individuals?
- Should we put some extra constraint in the "individuals" organism descriptor? For instance, a parasite's ``taxon_id`` could be given as a sample taken from a human, but should we accept that as an individual? 
- If we all expect the individual to be always a human, and you mention to add that node to its JSONs for it to be easily interpreted, would changing the name of the object from ``individual`` to ``human`` (or a similar synonym) have the same effect and avoid redundancy?

Reply:
> * Shall I add the whole organism descriptor node to the individual or just some parts (e.g. the `taxon_id` alone)?

I vote having the same block. It's ontologised, understandable, and interoperable.

> * Would there be an scenario in which we will receive data unrelated to humans as individuals?

In the near future, probably no. Long-term future, who knows? We do have this in the past, and we should give ourselves a mechanism to represent all EGA data correctly and accurately for the day when we do curation.

> * Should we put some extra constraint in the "individuals" organism descriptor? For instance, a parasite's `taxon_id` could be given as a sample taken from a human, but should we accept that as an individual?

This is probably a good idea. A flag to the user during validation step maybe if they don't choose human. "Did you really mean <blah> instead of human? Remember, EGA only accepts human data." This consideration is separate from this PR so file this away for an idea to implement with validation rules.

> * If we all expect the individual to be always a human, and you mention to add that node to its JSONs for it to be easily interpreted, would changing the name of the object from `individual` to `human` (or a similar synonym) have the same effect and avoid redundancy?

I like the original `individual` nomenclature. It's more in line with the term used in the scientific community for people as subjects in studies.

Reply:
DOne